package/skelcd-control-openSUSE.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu Mar 18 17:04:19 UTC 2015 - dvaleev@suse.com
+
+- Parse out non-oss repos for Power
```

Comment:
ok, so please add it to changes something like `Parse out non-oss repos for Power as it is not build for it` and add comment to spec file, so we know that we can remove if we start building for ppc